remote-state/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_s3_bucket" "remote_state" {
+    bucket = "terraform-remote-state-bl"
+}
+
+resource "aws_dynamodb_table" "terraform-remote-state-db" {
+  name           = "terraform-remote-state-db"
+  billing_mode   = "PROVISIONED"
+  read_capacity  = 1
+  write_capacity = 1
+  hash_key       = "LockID"
+
+
+  attribute {
+    name = "LockID"
+    type = "S"
+  }
+
+}
```

Comment:
could you please, add newline at the end of file.

Reply:
fixed